packages/application-shell/src/components/application-shell/application-shell.spec.js

```diff
@@ -182,11 +183,9 @@ describe('when user has no default project', () => {
       },
     });
     await waitFor(() => {
-      expect(window.location.replace).toHaveBeenCalledWith(
-        '/account/projects/new'
-      );
-      expect(rendered.queryByText('OK')).not.toBeInTheDocument();
+      expect(location.replace).toHaveBeenCalledWith('/account/projects/new');
     });
+    expect(rendered.queryByText('OK')).not.toBeInTheDocument();
```

Comment:
Any reason for moving this outside of the "wait"?

Reply:
Just to isolate things a bit. We waitFor something and then expect something outside. I am fine either way just felt it wasnt needed having both in it. For one also as I wasnt sure what would happen if B passes and A doesnt. Then A passes and B too. Feels a bit odd.

Reply:
The only concern I have is that the two expectations are not related to each other, meaning that given the first passes, the second one might not be "ready" yet.
That's why they are both in the "wait" block. Maybe we can have 2 waiting blocks?

Reply:
Oki doks. I agree. I changed it again :)